### PR TITLE
add NOTICE file with attribution of initial maven-wrapper module

### PR DESCRIPTION
**Pull request by hboutemy**
see initial commit from 2013 https://github.com/apache/maven-wrapper/commit/fa267d489f0950daf6825dcdd17ddb4de9ad9a20

### CODE REVIEW

**Review by jvanzyl — APPROVED** [review]

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Apache Maven Wrapper
+Copyright 2020-2022 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+The Initial Developers of maven-wrapper module, which is copied from, derived from,
+or inspired by Gradle Wrapper, are Hans Dockter and Adam Murdoch.
+Copyright 2007 the original author or authors.
```
Comment (marcphilipp) [inline NOTICE:9]: This makes it sound like Hans and Adam were the initial developers of the maven-wrapper module, i.e. as if they set out and wrote it for Maven.

```suggestion
The original idea and initial implementation of the maven-wrapper module is derived 
from the Gradle Wrapper which was written originally by Hans Dockter and Adam Murdoch.
Copyright 2007 the original author or authors.
```

**Review by marcphilipp — CHANGES_REQUESTED** [review]

**Review by marcphilipp — APPROVED** [review]

**Comment by marcphilipp** [comment]
@hboutemy Thanks for taking care of this! 🙂

**Comment by hboutemy** [comment]
being an OSS contributor, it's important to respect others work as much as I want my work to be respected :)
thank you for helping

**Review by britter — APPROVED** [review]
Thank you!

**Pull request merged**